snap/snapcraft.yaml

```diff
@@ -70,6 +79,7 @@ parts:
       - libqt5core5a
       - libqt5network5
```

Comment:
Remove all the `libqt` libraries. Newer versions are provided by the kde-frameworks content snap. Mixing libraries from different qt versions might break some things.

PS: you can see which libraries are provided via the commandline: `find /snap/kde-frameworks-5-qt-5-14-core18/current/ -name libQt5Core.so.5`. Websites such as https://ubuntu.pkgs.org/18.04/ubuntu-main-amd64/libqt5core5a_5.9.5+dfsg-0ubuntu1_amd64.deb.html can help you figure out what libraries a package installs.